packages/components/src/Chips/InternalChipSingleSelect.tsx

```diff
@@ -34,7 +35,22 @@ export function InternalChipSingleSelect({
               }}
               disabled={child.props.disabled}
             />
-            <InternalChip {...child.props} active={isSelected} />
+            <InternalChip
+              {...child.props}
+              {...(isSelected
+                ? {
+                    suffix: (
+                      <Icon
+                        size="small"
+                        name="checkmark"
+                        color="interactiveSubtle"
+                      />
+                    ),
+                  }
+                : {})}
+              active={isSelected}
+              tabIndex={undefined}
```

Comment:
Since the default for tab index is set to 0 when it is undefined this should probably be `-1` since the Checkbox / Radio in the internal chips will handle the focus of the chip.

Reply:
```suggestion
              tabIndex={-1}
```

Reply:
Not actually needed here so I will resolve this